modules/web/src/assets/config/changelog.json

```diff
@@ -1,17 +1,4 @@
 {
-  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.22.md#v22212",
-  "entries": [
-    {
-      "category": "added",
-      "description": "Add support for Kubernetes v1.24.17, v1.25.16 and v1.26.13 and set default version to v1.25.16."
-    },
-    {
-      "category": "changed",
-      "description": "Update EKS/AKS version matrix to only include Kubernetes versions supported by those managed offerings. For  AKS 1.26-1.28 are supported, for EKS 1.24 to 1.28. The default for newly created external clusters is now 1.28."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix a bug where Operating System Profiles were not being listed for GCP/GCE."
-    }
-  ]
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.22.md#v22213",
+  "entries": []
```

Comment:
I think there should be something in the changelog entries. Or all changes only related to backend?

Reply:
For this patch release, there are no changes that will be visible on the UI so we keep it empty.